kitsune/sumo/static/sumo/js/codemirror.sumo-hint.js

```diff
@@ -0,0 +1,38 @@
+(function(mod) {
+  if (typeof exports == "object" && typeof module == "object") // CommonJS
+    mod(require("../../lib/codemirror"));
+  else if (typeof define == "function" && define.amd) // AMD
+    define(["../../lib/codemirror"], mod);
+  else // Plain browser env
+    mod(CodeMirror);
+})(function(CodeMirror) {
```

Comment:
I suspect we could just assume that this code is running in a plain browser env, or switch to browserify and use CommonJS. Either way, we don't need to include all of them.